core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -161,48 +162,48 @@ class HttpGoogleServicesDAO(
 
   override def setupWorkspace(userInfo: UserInfo, project: RawlsBillingProject, policyGroupsByAccessLevel: Map[WorkspaceAccessLevel, WorkbenchEmail], bucketName: String, labels: Map[String, String]): Future[GoogleWorkspaceInfo] = {
 
-    def insertBucket: Map[WorkspaceAccessLevel, WorkbenchEmail] => Future[String] = { policyGroupsByAccessLevel =>
+    def insertBucket(): Future[String] = {
       implicit val service = GoogleInstrumentedService.Storage
-      retryWhen500orGoogleError {
-        () => {
-          // bucket ACLs should be:
-          //   project owner - bucket writer
-          //   workspace owner - bucket writer
-          //   workspace writer - bucket writer
-          //   workspace reader - bucket reader
-          //   bucket service account - bucket owner
-          val workspaceAccessToBucketAcl: Map[WorkspaceAccessLevel, String] = Map(ProjectOwner -> "WRITER", Owner -> "WRITER", Write -> "WRITER", Read -> "READER")
-          val bucketAcls =
-            policyGroupsByAccessLevel.map { case (access, policyEmail) => newBucketAccessControl(makeGroupEntityString(policyEmail.value), workspaceAccessToBucketAcl(access)) }.toSeq :+
-              newBucketAccessControl("user-" + clientEmail, "OWNER")
-
-          // default object ACLs should be:
-          //   project owner - object reader
-          //   workspace owner - object reader
-          //   workspace writer - object reader
-          //   workspace reader - object reader
-          //   bucket service account - object owner
-          val defaultObjectAcls =
-          policyGroupsByAccessLevel.map { case (_, policyEmail) => newObjectAccessControl(makeGroupEntityString(policyEmail.value), "READER") }.toSeq :+
-              newObjectAccessControl("user-" + clientEmail, "OWNER")
-
-          val logging = new Logging().setLogBucket(getStorageLogsBucketName(project.projectName))
-
-          val bucket = new Bucket().
-            setName(bucketName).
-            setAcl(bucketAcls.asJava).
-            setDefaultObjectAcl(defaultObjectAcls.asJava).
-            setLogging(logging).
-            setLabels(labels.asJava)
-          val inserter = getStorage(getBucketServiceAccountCredential).buckets.insert(project.projectName.value, bucket)
-          executeGoogleRequest(inserter)
-
-          bucketName
-        }
-      }
+
+      val logging = new Logging().setLogBucket(getStorageLogsBucketName(project.projectName))
+      val bucket = new Bucket().setName(bucketName).setLogging(logging).setLabels(labels.asJava)
+
+      val inserter = getStorage(getBucketServiceAccountCredential).buckets.insert(project.projectName.value, bucket)
+
+      retryWhen500orGoogleError { () =>
+        executeGoogleRequest(inserter)
+      }.map(_.getName)
```

Comment:
why are we still creating the bucket with the old client instead of using `googleStorageService`? The main reason why I care is that the logging is different. Create bucket will look different than the other api calls we make to setup the bucket which might be harder to figure out.

Reply:
I wanted to be surgical with this change but you bring up a good reason not to be. I'll use the google2 lib for creating the workspace bucket too.